app/Domain/Slack/Channels/SlackApiChannel.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Domain\Slack\Channels;
+
+use Illuminate\Http\Client\Response;
+use Illuminate\Notifications\Notification;
+use Illuminate\Support\Facades\Http;
+
+class SlackApiChannel
+{
+    public function send($notifiable, Notification $notification): Response
```

Comment:
Can you type `$notifiable`?

Reply:
In general, it can be anything that uses trait `Notifiable`

Reply:
Oh no, maybe you could add some interface fot that?